### PR TITLE
Add tutorial, fix #8

### LINKED ISSUE #8: Write tutorial which uses OnionBalance to load balance between multiple web servers
The current documentation is quite abstract and could to a better job of getting a new users up and running.

The tutorial should outline the full process for:
-  Setting up the web servers 
- Configure the back end Tor instances
- Creating an OnionBalance config file
- Running the OnionBalance management server with a supervisor process

### PR DESCRIPTION
**Pull request by FedericoCeratto**
Initial version.

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
+.. _tutorial:
+
+Tutorial
+========
+
+This is a step-by-step tutorial to help you configure OnionBalance.
+
+OnionBalance implements `round-robin` like load balancing on top of Tor
+onion services. A typical OnionBalance deployment will incorporate one management
+servers and multiple backend application sdoervers.
```
Comment (ypid) [inline docs/tutorial.rst]: Probably "servers" is meant instead of "sdoervers".

File: docs/tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+You want to run:
+
+- one or more OnionBalance processes, to perform load balancing, on hosts named obhost1, obhost2
+- two or more Tor processes, to run the Onion Services, on hosts named torhost1, torhost2
+- two or more servers (e.g. web servers) on hosts named webserver1, webserver2
+
+Scaling up:
+
+- the number of obhostX can be increased by this will not help handling more traffic
```
Comment (DonnchaC) [inline docs/tutorial.rst]: Typo "increased **but** this will not"

File: docs/tutorial.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+- the number of torhostX can be increased up to 10 instances to handle more traffic
+- the number of webserverX can be increased up to 60 to handle more traffic
```
Comment (DonnchaC) [inline docs/tutorial.rst]: @FedericoCeratto I'm not sure the distinction between the Tor host and webserver here. You can scale the HTTP traffic after Tor with a standard application level proxy such as haproxy.

OnionBalance can now use introduction points from 60 different Tor instances at one time. There can be 60 torhostX hosts and there is no limit on the number of web servers.

File: docs/tutorial.rst [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+If you have other obhostX:
+
+.. code-block:: bash
+
+   $ sudo apt-get install onionbalance
+   $ mkdir -p /var/run/onionbalance
+   $ chown onionbalance:onionbalance /var/run/onionbalance
+
+Copy /etc/onionbalance/*.key and /etc/onionbalance/config.yml from obhost1 to all hosts in obhostX
```
Comment (DonnchaC) [inline docs/tutorial.rst]: You need to escape `*`'s with a backslash in RST.

**Comment by FedericoCeratto** [comment]
Thanks, updated.

**Comment by DonnchaC** [comment]
Thanks @FedericoCeratto. I rebased and merged this with some changes in ee0246b.

**Pull request closed without merging**